app/javascript/src/Settings/styles/authentication.scss

```diff
@@ -2,7 +2,8 @@
   display: none !important;
 }
 
-#service_deployment_option_input {
+#service_deployment_option_input,
```

Comment:
Why not using same style of `inline-hints`?
![authentication](https://user-images.githubusercontent.com/13486237/67849533-02184a80-fb07-11e9-8ef3-64132b687d59.png)